libwebsockets.sh

```diff
@@ -23,11 +22,11 @@ cmake $SOURCEDIR/                                                   \
       -DLWS_WITH_STATIC=ON                                          \
       -DLWS_WITH_SHARED=OFF                                         \
       -DLWS_WITH_IPV6=ON                                            \
+      -DLWS_WITH_ZLIB=OFF                                           \
```

Comment:
Why?

Reply:
We don't use the data compression in websockets, hence no need for this dependency. But just today I had a discussion with @costing that we should enable the compression. Maybe we should keep zlib here for future.

Reply:
Returned zlib back to facilitate enabling the compression in future.

Reply:
Thanks. I think libwebsockets is also used somewhere else.

Reply:
I had to disable zlib support because of the build problems on ci machines. I suggest that I make a new pull request to enable zlib again later, but now merge this PR to have basic libjalien functionality.
libwebsockets is used only in JAliEn-ROOT, it won't affect other packages.